index.js

```diff
@@ -1,7 +1,7 @@
 /** @babel */
 const lazyReq = require('lazy-req')(require);
 
-const editorconfig = lazyReq('editorconfig');
```

Comment:
When you `lazyReq` something, it becomes a function that will require the library on demand.
```js
const editorconfig = lazyReq('editorconfig')
editorconfig().parse(file) //was editorconfig.parse(file)
```